### DISCUSSION TITLE
Is there support in Fastlane for App Icons?

### DISCUSSION THREAD

**Discussion by davidakoontz** [fastlane/fastlane#22217] [Ideas]
Seems like a way to upload the various App Icons would be a Fastlane tool.

Any support for Icon Apps - Color Themes etc?

**Comment by john589dane** [comment]
Hello,
Fastlane is indeed a powerful tool for managing your app's deployment processes, including uploading app icons. The Fastlane Plugin AppIcon can help generate all required icon sizes from a master application icon. This can streamline the process, saving you time and ensuring consistency across all icon sizes. [Your Texas Benefitsi com](https://www.yourtexasbenefitsi.com)


Best Regards,
Joan Dane
